### DISCUSSION TITLE
Using Vercel Postgres

### DISCUSSION THREAD

**Discussion by ajasinger** [vercel/community#2871] [Help] [answered]
### Summary

I am new to coding and have not set up a database before. I am building with Next.js and hosting on Vercel so would love to use Vercel Postgres for the database but since it's so new I can't find any tutorials online on how to build the database. Is it basically the same as using Supabase and Postgres -- would learning how to create a database using Supabase be directly transferrable to setting up a Vercel Postgres database? Thank you!

### Steps to Reproduce

N/A

**Comment by leerob** [comment, marked as answer]
Hey! Vercel Postgres is standard Postgres, so any Postgres tutorial should teach you everything you need to know about how to use it. Supabase is also just Postgres, so the knowledge would be interchangeable between services. Hope this helps!

**Reply by ajasinger** [reply]
Thank you!
